src/state/_common/recomputeHelpers.js

```diff
@@ -0,0 +1,6 @@
+import stringify from 'fast-stringify';
+
+// Use fast-stringify instead of default JSON.stringify to get cache key
+const serialize = args => stringify(args);
+
+export const recomputeSelectorOptions = {serialize};
```

Comment:
'recomputeSelectorOptions' to me says some object / constant, not a function transforming input

Reply:
I just use the terminology from the documentation https://www.npmjs.com/package/@jvitela/recompute

Reply:
Eh, my bad. I can't read.